src/main/java/com/google/sps/servlets/getArticlesServlet.java

```diff
@@ -14,14 +14,14 @@
 
 package com.google.sps.servlets;
 
+import java.util.*;
+import java.lang.*;
+import java.time.Duration;
```

Comment:
During the first few weeks, I remember importing * for a java.util, and David said a better practice was to important specifically the ones we are using.

Reply:
+1

Reply:
Also the java.lang package doesn't need to be imported since its done automatically by java.

Reply:
Will make those changes now! :)